contracts/modules/capital/Pool.sol

```diff
@@ -137,7 +140,14 @@ contract Pool is IPool, MasterAwareV2, ReentrancyGuard {
         continue;
       }
 
-      total += getAssetValueInEth(assets[i].assetAddress);
+      address assetAddress = assets[i].assetAddress;
+      uint amountInSwapOperator;
+
+      if (isAssetInSwapOperator(i, _assetsInSwapOperatorBitmap)) {
+        amountInSwapOperator = assetsInSwapOperator[assetAddress];
+      }
```

Comment:
style: might be more concise and readable to make this ternary as we'll explicitly assign 0.

```solidity
      uint amountInSwapOperator = isAssetInSwapOperator(i, _assetsInSwapOperatorBitmap)
        ? assetsInSwapOperator[assetAddress]
        : 0;
```